### ISSUE TITLE
Remove JQuery dependency for Highcharts

### ISSUE THREAD

**Issue by vincent-dm** [cljsjs/packages#510]
The Highcharts library used to depend on JQuery, but this is not the case anymore ([in 2013 they added a standalone framework](http://www.highcharts.com/articles/2-news/58-highcharts-standalone-framework) as an alternative to JQuery, but since Dec 2015 even that is not needed anymore).

The old (JQuery) way to create a chart in cljs:
```clojure
(.highcharts (js/$ (.getElementById js/document "id")) (clj->js chart-config)))
```

However, when the standard Highcharts library is loaded in the browser, the Highcharts.Chart method can attach a chart into the DOM without using JQuery.
An example:
```clojure
(js/Highcharts.Chart. (.getElementById js/document "id") (clj->js chart-config))
```

Since JQuery is a pretty hefty dependency, it would be good to remove it. I'm aware that most examples of Highcharts still use JQuery syntax, but since cljs-users already have to re-write every code example, I don't see why we should keep relying on JQuery.

Thanks!

**Comment by vincent-dm** [comment]
@martinklepsch do you think we can drop jquery?

**Comment by ducky427** [comment]
@vincent-dm, would excluding `cljsjs/jquery` from your dependency list in `project.clj` work? See [this](https://github.com/technomancy/leiningen/blob/master/sample.project.clj#L48) for more details on how to do it.

(admitting that a new release would be a better option)

**Comment by yatesco** [comment]
I didn't realise this so my PR to update to HC4.2.5 (https://github.com/cljsjs/packages/issues/585) still lists the jquery dependency.

**Comment by Deraen** [comment]
Merged in #586

**Comment by vincent-dm** [comment]
Thanks!
